scvi/hub/hub_metadata.py

```diff
@@ -0,0 +1,188 @@
+import json
+from dataclasses import dataclass
+from typing import List, Optional
+
+import torch
+from dataclasses_json import dataclass_json
+from huggingface_hub import ModelCard, ModelCardData
+
+from ._constants import (
+    ANNDATA_VERSION_TAG,
+    ANNOTATED_TAG,
+    DEFAULT_MISSING_FIELD,
+    DEFAULT_NA_FIELD,
+    DEFAULT_PARENT_MODULE,
+    HF_LIBRARY_NAME,
+    MODALITY_TAG,
+    MODEL_CLS_NAME_TAG,
+    SCVI_VERSION_TAG,
+    TISSUE_TAG,
+)
```

Comment:
maybe use an enum? named tuple?

Reply:
In the interest of time, I'm bundling some of these less pressing comments in an issue to address in a separate pr.
https://github.com/scverse/scvi-tools/issues/1817